src/main/command/statuscheckcommands/StatusCheckCommand.ts

```diff
@@ -10,14 +11,18 @@ export class StatusCheckCommand extends Command {
                    userPerms: Permissions,
```

Comment:
Can you write a short summary of what the class does and the execute methods? Refer to the other execute methods in other command classes to see what I mean.